src/rviz_visual_tools_demo.cpp

```diff
@@ -410,25 +410,29 @@ class RvizVisualToolsDemo
     ROS_INFO_STREAM_NAMED(name_, "Displaying ABCD Plane");
     double x_width = 0.15;
     double y_width = 0.05;
-    pose1 = Eigen::Isometry3d::Identity();
-    y += space_between_rows;
-    pose1.translation().y() = y;
-    pose1.translation().x() = width;
 
     Eigen::Vector3d n;
     double A, B, C, D;
+    y += space_between_rows;
+    double x_plane = 0, y_plane = y;
+
+    pose1 = Eigen::Isometry3d::Identity();
+    pose1.translation().x() = x_plane;
+    pose1.translation().y() = y_plane;
+    publishLabelHelper(pose1, "ABCD Plane");
 
     for (std::size_t i = 0; i < 10; ++i)
     {
-      A = i * step;
-      B = pose1.translation().y();
+      x_plane = i * step;
+      pose1.translation().x() = x_plane;
+      A = x_plane;
+      B = y_plane;
```

Comment:
y, y_plane is quite big. if you use this for computing D like you did, then the distance will be huge

Reply:
I didn't follow what you mean here @henningkayser but the rest looked good to me during my test.

Reply:
@mcevoyandy I'm talking about the effect seen in @AndyZe's screenshots above (https://github.com/PickNikRobotics/rviz_visual_tools/pull/120#discussion_r286626636).